src/reynir_correct/main.py

```diff
@@ -341,17 +343,43 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
             )
             for ann in a
         ]
+        if args.text:
+            arev = a
+            arev.sort(key=lambda ann: (ann.start, ann.end), reverse=True)
+            if sent.tree is None:
+                # No need to do more, no grammar errors have been checked
+                print(cleaned, file=args.outfile)
```

Comment:
But how about context-free token-level errors, which may have been corrected even if the sentence did not parse?

Reply:
They are fixed in `sent = check_tokens(toklist)`.